models/catalog/v1alpha1/catalog.go

```diff
@@ -80,33 +81,59 @@ const (
 	Community ContentClass = "community"
 )
 
-func (c ContentClass) String() string {
-	switch c {
-	case Official:
-		return "official"
-	case Verified:
-		return "verified"
-	case Community:
-		fallthrough
-	default:
-		return "community"
+// Load and extract descriptions and enum values from schema
+func loadSchemaDescriptions() (map[string]ContentClassObj, error) {
```

Comment:
Meshery Schemas are to be imported as released golang packages. 

I'm trying to understand why this approach is necessary.

Reply:
Yes we are importing them as golang package 

https://github.com/meshery/meshkit/pull/542/files#diff-6aee5a74826065e2f1c176431a0d3f7274d9baac2a66d75d97cf1efad6c452aaR8

This part extracts the class description from the schema which we get from released schema golang package, we can change this func name to a better descriptive name like `extractClassAndDescriptions `

Reply:
Yes we are importing them as golang package

https://github.com/meshery/meshkit/pull/542/files#diff-6aee5a74826065e2f1c176431a0d3f7274d9baac2a66d75d97cf1efad6c452aaR8

This part extracts the class description from the schema which we get from released schema golang package, we can change this func name to a better descriptive name like extractClassAndDescriptions

Reply:
Thanks. I'm sorry to say that I still don't understand the need for a function specifically for this property and not either a general utility function to retrieve property values from a separate, associated property.

Perhaps, we shouldn't have descriptions in an array to match an enum in a separate property, but use an approach like:

```
{
    "type": "string",
    "oneOf": [
        { "const": "community", "description": "description of the class." },
        { "const": "verified", "description": "another description of the class" },
        { "const": "official", "description": "Best" },
    ]
}
```

Reply:
Enhancement proposed - https://github.com/meshery/schemas/issues/109

Reply:
@shubham251972 will you be taking care of this?

Reply:
JSON schema updated with oneOf - https://github.com/meshery/schemas/pull/126/files

Updated meshkit code to take benefit of `oneOf` and return classes